src/components/ProcessCreate/CostPreview.tsx

```diff
@@ -26,6 +26,7 @@ import { TbDatabaseExclamation } from 'react-icons/tb'
 import { useClaim } from '~components/Faucet/Claim'
 import { useProcessCreationSteps } from './Steps/use-steps'
 import imageHeader from '/assets/voc-tokens.jpg'
+import { signinUrlParams } from '~components/Faucet/use-faucet'
```

Comment:
Should be properly sorted. Recommended to use the sort on save option of vscode:

~~~json
"editor.formatOnSave": true,
"editor.defaultFormatter": "esbenp.prettier-vscode",
"editor.codeActionsOnSave": {
  "source.organizeImports": true
}
~~~